src/graph.ts

```diff
@@ -477,7 +588,7 @@ export class FactoryGraph {
                 console.log(container)
                 throw new Error("Container exceeds outgoing link limit")
             }
-            if (container.egress > container.ingress) {
+            if (container.egress > 1.0001 * container.ingress) {
```

Comment:
Do we run into issues with float point numbers here?

Reply:
Ah yes, I forgot to mention this change. There is probably a more elegant way to handle this.

Reply:
Fix in https://github.com/lgfrbcsgo/du-factory-generator/commit/5cc1a153e809752d2a07f246b0845d258a5da174